OpenRA.Mods.Common/Widgets/Logic/Settings/HotkeysSettingsLogic.cs

```diff
@@ -322,7 +322,8 @@ void OverrideHotkey()
 		bool IsHotkeyVisibleInFilter(HotkeyDefinition hd)
 		{
 			var filter = filterInput.Text;
-			var isFilteredByName = string.IsNullOrWhiteSpace(filter) || hd.Description.Contains(filter, StringComparison.OrdinalIgnoreCase);
+			var isFilteredByName = string.IsNullOrWhiteSpace(filter) ||
+				hd.Description.Contains(filter, StringComparison.CurrentCultureIgnoreCase);
```

Comment:
Changing from `OrdinalIgnoreCase` to `CurrentCultureIgnoreCase` is intentional here because of user input?

Reply:
Yes, since the filter text is user entered I chose to apply the user's culture.

Reply:
The description might not be in the user's culture though. I'm not sure if that makes a difference/an impact. (Feel free to resolve after reading.)

Reply:
I dont have a real answer for that. Hopefully using the user's culture is at least better than not? https://learn.microsoft.com/en-us/dotnet/standard/base-types/best-practices-strings#choosing-a-stringcomparison-member-for-your-method-call has "Most user input." mapped to CurrentCulture, so that's still my best call.